home/WEB-INF/jsp/publication/publication-image-gallery.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ include file="/WEB-INF/jsp-include/tag-import.jsp" %>
+<%@ page import="org.zfin.properties.ZfinPropertiesEnum" %>
+
+    <style>
+        #xpresimg_control_box {
+            margin-top: 20px;
+        }
+    </style>
+
+    <c:if test="${!empty imageResults}">
+        <div id="xpresimg_all">
+            <input type="hidden" name="xpatsel_thumbnail_page" id="xpatsel_thumbnail_page_hidden_field" value="1"/>
+            <div id="xpresimg_control_box">
+                <span id="xpresimg_thumbs_title">Figure Gallery</span>
+                <span id="xpresimg_controls"></span>
+            </div>
+            <div id="xpresimg_box"></div>
+            <div id="imagebox_maxnote" style="display: none;"></div>
+            <div id="xpresimg_imagePreload"></div>
+        </div>
+
+        <script type="text/javascript">
+            var imageBox = new ImageBox();
```

Comment:
I would tend to put this into a JS file (maybe a react component). What do you think of that idea?

Reply:
Good idea, though, I did not want to spend too much time on this functionality. Please create a new ticket if you feel strong about this refactor.